python/ray/_private/ray_client_microbenchmark.py

```diff
@@ -94,12 +94,7 @@ def actor_concurrent():
 def main(results=None):
     results = results or []
 
-    ray_config = {
-        "_system_config": {
-            "put_small_object_in_memory_store": True
-        },
-        "logging_level": logging.WARNING
-    }
```

Comment:
Note that this will change the Ray client microbenchmark results as well, but we can just make a note of that in our release process.